dev-python/scanpy/scanpy-1.11.3.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=hatchling
+PYTHON_COMPAT=( python3_{11..13} )
+
+inherit distutils-r1 pypi
+
+DESCRIPTION="Single-cell analysis in Python (scalable to >1M cells)"
+HOMEPAGE="https://github.com/scverse/scanpy"
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	>=dev-python/anndata-0.9[${PYTHON_USEDEP}]
+	>=dev-python/h5py-3.8[${PYTHON_USEDEP}]
+	dev-python/joblib[${PYTHON_USEDEP}]
+	>=dev-python/legacy-api-wrap-1.4.1[${PYTHON_USEDEP}]
+	>=dev-python/matplotlib-3.7[${PYTHON_USEDEP}]
+	dev-python/natsort[${PYTHON_USEDEP}]
+	>=dev-python/networkx-2.7[${PYTHON_USEDEP}]
```

Comment:
```suggestion
	>=dev-python/networkx-2.7.1[${PYTHON_USEDEP}]
```

Reply:
Concerning the versioned dependencies: I'm wondering if it makes sense to add the versions here at all, especially because the only version of `networkx::gentoo` is `3.5-r1`, which easily satisfies this dependency.

I'm thinking of removing all dependency versions, as long as there is no older version still available in the gentoo (or science) trees.

Does that make sense?

Reply:
I think adding them does not hurt. Technically, but very unlikely somebody could run a system without updates for >1 year with the old versions still installed.

Reply:
I've now added the exact deps for the release and ordered them the same as pyproject.toml for easier matching/updating